app/controllers/votes_controller.rb

```diff
@@ -2,22 +2,23 @@ class VotesController < ApplicationController
   def create
     @vote = Vote.new(vote_params.merge(user: current_user))
     if @vote.save
-      @count = Vote.where(article_id: params[:article_id]).sum(:value)
-      render json: {id: params[:article_id].to_i, vote: @count, path_up: votes_path(article_id: params[:article_id], value: 1), 
-        path_down: votes_path(article_id: params[:article_id], value: 1)}
+      @count = Vote.where(voteable_id: params[:voteable_id], voteable_type: params[:voteable_type]).sum(:value)
+      
+      render json: {id: params[:voteable_id].to_i, vote: @count, path_up: votes_path(voteable_id: params[:voteable_id], voteable_type: params[:voteable_type],value: 1), 
```

Comment:
there are a lot of variables being passed here. Could you put them on new lines to help with readability?
eg

```ruby
render json: {
  id: params[:voteable_id].to_i, 
  vote: @count, 
  path_up: votes_path(
    voteable_id: params[:voteable_id], 
    voteable_type: params[:voteable_type], 
    value: 1
  ),
  path_down: votes_path(
    voteable_id: params[:voteable_id], 
    voteable_type: params[:voteable_type], 
    value: -1
  )
}
```